### ISSUE TITLE
Doom installation failure

### ISSUE THREAD

**Issue by Midnight1938** [catppuccin/emacs#207]
I am trying to install Catppuccin via Doom emacs. But after sync, the theme does not show up in the "picker".

Adding it to initialize at startup in config with `(setq doom-theme 'catppuccin)` also does nothing

**Comment by jtbx** [comment]
I'm not sure how doom works with themes but can you try `(load-theme 'catppuccin t)`?

**Comment by jtbx** [comment]
That's strange, you could ask one of the doom developers about this. I can't help you here I'm sorry.

**Comment by MartV0** [comment]
Can confirm there seems to be some issue with doom emacs and this theme. Me entire doom emacs broke, removing the catpuccin theme fixed it

**Comment by jtbx** [comment]
@MartV0 you're looking for #208. Try loading the theme in `after-init-hook`.

**Comment by sgoudham** [comment]
Has this now been resolved?

**Comment by jtbx** [comment]
No, I couldn't find the cause. Might be something to do with catppuccin-definitions.el.

@Midnight1938 Can you still reproduce this and if so can you check the messages buffer?

**Comment by MrMic** [comment]
cannot use the theme either with doom emacs ☹️

**Comment by AliGhahraei** [comment]
@MrMic what have you tried? I got it working with this in `config.el` (what @jtbx [suggested](https://github.com/catppuccin/emacs/issues/208#issuecomment-2585962403), but with `use-package!`):

```elisp
(use-package! catppuccin-theme
  :init (setq catppuccin-flavor 'mocha)
  :hook (after-init . (lambda () (load-theme 'catppuccin))))
```

**Comment by MrMic** [comment]
Yes indeed, I got it working! Thanks.
BTW, for those, like me, who use Doom Emacs, here is config:
- In `init.el`:
`(package! catppuccin-theme
  :recipe (:host github :repo "catppuccin/emacs"))
`
- In `config.el`:
`(use-package! catppuccin-theme
  :init (setq catppuccin-flavor 'mocha))
`

KR
